css/files.css

```diff
@@ -0,0 +1,75 @@
+div[data-cy-files-list].files-list {
+	.files-list__row {
+		&:hover {
+			background-color: var(--ion-color-blue-b1);
+		}
+		&.active {
+			background-color: var(--ion-color-blue-b1);
+		}
+		.files-list__row-icon {
+			color: var(--ion-color-blue-b4);
+		}
+		.files-list__row-actions {
+			.action-items {
+				color: var(--ion-color-blue-b4);
+			}	
+			.icon-vue {
+				color: var(--ion-color-blue-b4);
+			}
+			.material-design-icon{
+				color: var(--ion-color-blue-b4);
+			}
+		}
+	}
+
+	/* mobile and tablet */
+	@media only screen and (max-width: 1199px) {
+		.files-list__filters {
+			height: auto;
+		}
+	}
+}
+
+div.v-popper--theme-dropdown.v-popper__popper:has(.v-popper__wrapper) {
+	border-radius: 16px;
+	filter: none;
+
+	div.v-popper__wrapper {
+		border-radius: inherit;
+		border: 1px solid var(--ion-color-blue-b4);
+
+		div.v-popper__inner {
+			padding: 2px 12px;
+			background-color: var(--color-main-background);
+
+			li.action:has(button.action-button) {
+				&.active {
+					background-color: var(--ion-color-blue-b1);
+				}
+
+				button.action-button {
+					border-radius: 8px;
+					color: var(--ion-color-secondary);
+					font-style: normal;
+					font-weight: 600;
+					margin-top: 12px;
+					margin-bottom: 12px;
+
+					&:active {
+						background-color: var(--ion-color-blue-b2);
+					}
+
+					&:disabled {
+						color: var(--ion-color-cool-grey-c4);
+						background-color: var(--color-main-background);
+						cursor: not-allowed;
+					}
```

Comment:
The active state when clicking a button in the menu now has a border radius because of css/button.css L3